### ISSUE TITLE
string to LUUID

### ISSUE THREAD

**Issue by mrumaiz** [cdimascio/uuid-mongodb#7]
how to convert to LUUID from String? Is it possible?

**Comment by cdimascio** [comment]
@mrumaiz by LUUID, are you referring to Binary.SUBTYPE_UUID_OLD?

**Comment by vainamov** [comment]
> @mrumaiz by LUUID, are you referring to [Binary.SUBTYPE_UUID_OLD](http://mongodb.github.io/node-mongodb-native/3.2/api/Binary.html#.SUBTYPE_UUID_OLD)?

Yes he is. I believe in MongoDB 4.2 they changed the previous version of UUID to Legacy UUID (LUUID) and renamed the subtype to `SUBTYPE_UUID_OLD`. So currently trying to parse those UUIDs will fail at https://github.com/cdimascio/uuid-mongodb/blob/master/lib/index.js#L31
